docs/tools/debugging.md

```diff
@@ -319,6 +319,33 @@ If this is the first time you've looked, there may be other errors visible which
 
 If you're not familiar with UNIX, this may seem like a lot of effort. It's a lot less effort than *guessing* your way through a problem though!
 
+## Debugging Javascript issues
+
+**Symptoms**
```

Comment:
Can you use headings here instead of bold text? (and for the other instances below as well)